contracts/bondingcurve/BondingCurve.sol

```diff
@@ -107,6 +107,18 @@ contract BondingCurve is IBondingCurve, OracleRef, PCVSplitter, Timed, Incentivi
         return token.balanceOf(address(this));
     }
 
+    /// @notice display the related token of the balance reported
+    /// @dev to allow compatibility with the Collateralization Oracle even though this contract isn't an IPCVDeposit
+    function balanceReportedIn() public view returns (address) {
+        return address(token);
+    }
+
+    /// @notice returns a manipulation resistant account of both the balance of underlying and protocol owned FEI
+    /// @dev to allow compatibility with the Collateralization Oracle even though this contract isn't an IPCVDeposit
+    function resistantBalanceAndFei() public view returns(uint256, uint256) {
+      return (balance(), 0);
```

Comment:
I wonder if protocol contracts should report their current balance of FEI instead of 0. Like there could be a weird attack where FEI is sent to the protocol contracts, and the protocol believes it is undercollateralized & mint TRIBE, where actually it is holding the FEI. At first glance this looks like an attack that is a loss for the attacker, but idk there may be some wierd interaction somewhere ?

Reply:
+1 this is definitely something to put some thought into - though, a special version of the Bonding Curve that implements logic susceptible to this kind of attack could just override this method.

If anything, it's better to leave it as the default of zero -  I cold see an attack far more likely where someone sends FEI borrowed from a lending pool to the contract, has the contract count it, and double-counts PCV this way.

Reply:
Default of 0 makes most sense in this context because the FEI is likely held as a deposit receipt in Fuse or whatever. Alternatively we could add a distinct "Fei" PCV deposit that reports in-line with expectations for FEI